app/validators/answers/option_answer_validator.py

```diff
@@ -90,21 +88,6 @@ def validate_duplicate_options(self):
             labels.add(option["label"])
             values.add(option["value"])
 
-    def validate_labels_and_values_match(self):
-        for option in self.options:
-            if "text_plural" in option["label"]:
-                continue
-
-            if isinstance(option["label"], str):
-                label = option["label"]
-            else:
-                label = option["label"]["text"]
-
-            if label != option["value"]:
-                self.add_error(
-                    self.ANSWER_LABEL_VALUE_MISMATCH, label=label, value=option["value"]
-                )
-
```

Comment:
What's the reason for removing this code?

Reply:
This is one of the requirements on the card, to remove validation that enforces that answer values match answer labels. Basically it is to cover this part of the design proposal:
`For answers with options, an optional answer_value must be provided to allow mapping to a single option value. The validation that enforces the label must match the value will be removed to prevent downstream mappings from changing when the label changes.`